exampleCourse/questions/demo/workspace/xtermjs/info.json

```diff
@@ -18,8 +18,9 @@
         "port": 8080,
         "home": "/home/student",
         "gradedFiles": [
-            "starter_code.h",
-            "starter_code.c"
+            "**/*.h",
+          "**/*.c",
+          "../**/*"
```

Comment:
Was this supposed to be committed?

Reply:
The first two yes, the last one was added for testing. Fixed in 45be3985a.